### PR TITLE
docs(externals): add array reference

### LINKED ISSUE #3503: (docs): externals array still referenced
This PR https://github.com/webpack/webpack.js.org/pull/2100 didn't remove all references to array in externals.
https://github.com/webpack/webpack.js.org/blob/2eb1c0e3c355a52a10cc54dba2dab30958fb7758/src/content/configuration/externals.md#array
https://webpack.js.org/configuration/externals/#array
https://github.com/webpack/webpack.js.org/blob/2eb1c0e3c355a52a10cc54dba2dab30958fb7758/src/content/configuration/externals.md#combining-syntaxes
https://webpack.js.org/configuration/externals/#combining-syntaxes
Which is confusing to users, as it seems array is still supported (but doesn't seem to be)

### PR DESCRIPTION
**Pull request by snitin315**
_describe your changes..._

closes #3503 

Reverts #2100

Added externals array references.

### CODE REVIEW

**Comment by montogeek** [comment]
If I understood correctly, this is still supported: https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json#L447

**Comment by snitin315** [comment]
@montogeek then I should revert the changes in #2100.

**Comment by EugeneHlushko** [comment]
All right, this needs testing and if it works revert, if it doesnt, file an issue in webpack repo

**Comment by snitin315** [comment]
@EugeneHlushko Tested locally, the array is still supported. 

**conf-**
 
Screenshot from 2020-03-21 20-14-30

Output-

Screenshot from 2020-03-21 20-15-22


Screenshot from 2020-03-21 20-13-02

**Comment by snitin315** [comment]
This PR reverts  #2100

**Comment by snitin315** [comment]
Friendly ping @montogeek

File: src/content/configuration/externals.md [hunk 1]
```diff
@@ -11,14 +11,15 @@ contributors:
...
+  - snitin315
 ---
 
 The `externals` configuration option provides a way of excluding dependencies from the output bundles. Instead, the created bundle relies on that dependency to be present in the consumer's (any end-user application) environment. This feature is typically most useful to __library developers__, however there are a variety of applications for it.
 
 
 ## `externals`
 
-`string` `object` `function`  `RegExp`
+`string` `array` `object` `function`  `RegExp`
```
Comment (EugeneHlushko) [inline src/content/configuration/externals.md]: We are in a habit of specifiyng what's allowed as array value, therefore i suggest going for:

```suggestion
`string` `[string, RegExp, function => string]` `object` `function => string`  `RegExp`
```

**Comment by snitin315** [comment]
@EugeneHlushko Done :+1:

File: src/content/configuration/externals.md [hunk 2]
```diff
@@ -194,7 +195,7 @@ module.exports = {
         amd: 'lodash',
         root: '_' // indicates global variable
       },
-      // Array
+      // [String]
```
Comment (EugeneHlushko) [inline src/content/configuration/externals.md]:
```suggestion
      // [string]
```

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**